app/models/event_log.rb

```diff
@@ -21,6 +21,7 @@ class EventLog < ActiveRecord::Base
   EMAIL_CHANGE_INITIATED = "Email change initiated"
   EMAIL_CHANGE_CONFIRMED = "Email change confirmed"
   TWO_STEP_ENABLED = "2-step verification enabled"
+  TWO_STEP_DISABLED = "2-step verification disabled"
```

Comment:
It could, but on reflection it might be better to set the `initiator` so that the actual super admin who initiated the action was logged as part of the event?